mbed_tools_ci_scripts/utils/aws_helpers.py

```diff
@@ -0,0 +1,117 @@
+"""Helpers for interacting with AWS services such as S3.
```

Comment:
This is not used by any common CI stuff, just by `mbed-targets`. I'd rather not have it in here until this is used by more than one package.

Reply:
Well it is a CI utility and I rather have them all in one place. Plus, it is not really mbed-targets specific